CenterForOpenScience/osf.io
# [No ticket] Project overview: Collection info

## Purpose
- Address small changes for Project Overview's Collection info area before development

## Changes
- A11y
  -  Use semantic `<button>` for toggling collection specific metadata and preprint's supplemental material dropdown
  - Added aria-label, as this button has no visible text
  - Styling
- Show character-lab specific metadata: school-type and study-design

## Screenshot
Before: missing school type and study design
![image](https://user-images.githubusercontent.com/51409893/194944319-c37caa70-116b-4716-a25a-252e1e33fa59.png)

After: school type and study design included
![image](https://user-images.githubusercontent.com/51409893/194944513-7176a2e6-9a04-4ead-9cf0-400594de427d.png)

Better accessibility for collections metadata dropdown and preprint supplemental materials dropdown
![image](https://user-images.githubusercontent.com/51409893/194942648-1c9de615-e0e1-4151-8d6d-8a9cd981cdb6.png)

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify that projects submitted to CharacterLab (or really any collection-provider that has options for "School Type" and "Study Design" show this information in the project overview page where is says "Included in CharacterLab's Collection"
- Verify that the button to toggle the collection-specific metadata passes A11y standards

What are the areas of risk?
- Just A11y related issues maybe?

Any concerns/considerations/questions that development raised?

## Documentation
- NA

## Side Effects
- I addressed another dropdown for Preprints that link to a given project for supplemental materials as well that could use better a11y

## Ticket
NA
